scripts/main.js

```diff
@@ -48,3 +48,23 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   })
 })
+
+
+
+//Capitalize first letter of First Name and Last Name
+const firstName = document.querySelector('#icon_prefix');
+const lastName = document.querySelector('#last_name');
+
+const capitalizeFirstLetter = (e) => {
+  e.target.value = e.target.value.charAt(0).toUpperCase() + e.target.value.slice(1);
+};
+
+firstName.addEventListener('keyup', capitalizeFirstLetter);
```

Comment:
The use of event listeners to Capitalize the first letter of the input fields is a wise decision. But the event should be on 'change' because keyup will be triggered each time the user presses any key. Now imagine how many times it will be triggered when the name 'Nebuchadnezzer' is typed. lol ;)